CMakeLists.txt

```
@@ -90,14 +90,54 @@ set(ScriptDirectory "${PROJECT_SOURCE_DIR}/scripts")
 set(ModuleBuilderScript "${ScriptDirectory}/module_builder.py")
 set(ModuleHeaderScript "${ScriptDirectory}/module_header.py")
 
+# Dynamics type. Defaults to DG1.
+# Available options are:
+# DG1 Discontinuous Galerkin, degree 1 (3 components)
+# DG2 Discontinuous Galerkin, degree 2 (6 components)
+set(isDG FALSE)
+
+# Replace a missing dynamics type with DG1
+if (NOT DynamicsType)
+    message("Defaulting dynamics to DG1")
+    set(DynamicsType "DG1")
```

```suggestion
    set(DynamicsType "DG2")
```

I think we'll most often run the model like this, but I'll admit that's not clear yet.

---

Agreed on both counts.